src/definitions/modules/dropdown.js

```diff
@@ -992,7 +992,7 @@ $.fn.dropdown = function(parameters) {
               module.clear();
             }
             module.debug('Creating dropdown with specified values', values);
-            module.setup.menu({values: values});
+            module.setup.menu({[[fields.values]]: values});
```

Comment:
Why double `[[` `]]`? Please reduce to only one

Reply:
Damn, a stupid mistake.
I reverted the commit in the end, because it did not fix all cases.
Added another commit that should fix all call paths.